webapp/src/modules/storage/middleware.js

```diff
@@ -13,25 +14,54 @@ import {
 } from 'modules/translation/actions'
 import { UPDATE_DERIVATION_PATH } from 'modules/wallet/actions'
 
-export function createStorageMiddleware(storageKey) {
-  const storageEngine = filter(createStorageEngine(storageKey), [
-    'transaction',
-    'translation',
-    ['wallet', 'data', 'locale'],
-    ['wallet', 'data', 'derivationPath']
-  ])
+export function createMainStorageMiddleware() {
```

Comment:
I decided to create a function for each piece of the store to be stored ( `createMainStorageMiddleware` and `createTransactionStorageMiddleware` ) because using the generic `createStorageMiddleware` felt a bit too much  code for the store, but I wonder if this is too much coupling for this file.